'raw' fails in CI with "cp: cannot stat 'recovery/usr/share/grub2/x86_64-efi/*.mod': No such file or directory"

<!-- Thanks for helping us to improve cos-toolkit! We welcome all bug reports. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->

**cos-toolkit version:**
<!-- Provide the version of system/cos or git checkout of cos-toolkit -->
master branch at e411d8b3f0044edffc6fafa39f3097b471ef46bc
**CPU architecture, OS, and Version:**
<!-- Provide the output from "uname -a"  -->

CI run: https://github.com/rancher-sandbox/cOS-toolkit/runs/2856393413?check_suite_focus=true

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

'raw-img' fails with

```
Run # Apparently having other repositories enabled causes luet to hang
removed '/etc/luet/repos.conf.d/luet.yml'
removed '/etc/luet/repos.conf.d/mocaccino-extra-stable.yml'
removed '/etc/luet/repos.conf.d/mocaccino-repository-index.yml'
/__w/cOS-toolkit/cOS-toolkit/images/img-builder.sh
 Downloading file repository.yaml from build


⠈  Downloading file tree.tar.zst from build
 Downloading file repository.meta.yaml.tar.zst from build
 🏠  Repository cOS revision: 1 - 2021-06-18 07:49:05 +0000 UTC
 ℹ Repository: cOS Priority: 0 Type: disk
WWW
 No packages to install
cp: cannot stat 'recovery/usr/share/grub2/x86_64-efi/*.mod': No such file or directory
make: *** [make/Makefile.raw:55: raw_disk] Error 1
Error: Process completed with exit code 2.
```

**To Reproduce**
<!-- Steps to reproduce the behavior, including the luet command used -->

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

**Logs**
<!-- If applicable, add logs with the "--debug" flag enabled to help explain your problem. -->

**Additional context**
<!-- Add any other context about the problem here. -->

--- a/manifest.yaml
+++ b/manifest.yaml
@@ -22,3 +22,4 @@ label: "COS_LIVE"
 build:
  - utils/nerdctl
  - utils/k9s
+ - recovery/cos

